README.md

```diff
@@ -1 +1,13 @@
 # hw01
+
+##*Ta'er*, Seer of Paths, Traverser of Seas
```

Comment:
I had trouble with this at first too! When formatting a header I believe there is supposed to be a space after the # or the header will not format correctly!